### PR TITLE
Make GUI examples more Pythonic

### PR DESCRIPTION
**Pull request by tildebyte**

### CODE REVIEW

File: mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+Move the scrollbars left and right to change the positions of the images.
 """
+from hscrollbar import HScrollbar
 
 def setup():
+    global hs1, hs2, img1, img2
     size(640, 360)
     noStroke()
     # Two scrollbars
-    global hs1, hs2
```
Comment (jdf) [inline mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde]: What's wrong with the global statements at point of use?

File: mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde [hunk 2]
```diff
@@ -35,51 +35,3 @@ def draw():
     hs2.display()
     stroke(0)
     line(0, height / 2, width, height / 2)
-
-
-class HScrollbar(object):
```
Comment (jdf) [inline mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde]: What's wrong with the inline class definitions?

Comment (tildebyte) [inline mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde]: I prefer breaking classes out into their own file, but that's just me.

File: mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde (hunk 1 above)
Comment (tildebyte) [inline mode/examples/Topics/GUI/Scrollbar/Scrollbar.pyde]: See my comment in #132

**Comment by tildebyte** [comment]
@jdf Done

**Pull request merged**